hadoop-hdds/docs/themes/ozonedoc/static/swagger-resources/recon-api.yaml

```diff
@@ -229,6 +233,229 @@ paths:
+          in: query
+          description: Only returns the limited number of results. The default limit is 1000.
+          required: false
+          schema:
+            type: integer
+            default: 1000
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  totalCount:
+                    type: integer
+                    example: 5
+                  buckets:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        volumeName:
+                          type: string
+                          example: vol1
+                        bucketName:
+                          type: string
+                          example: buck1
+                        versioning:
+                          type: boolean
+                          example: false
+                        storageType:
+                          type: string
+                          example: DISK
+                        creationTime:
+                          type: integer
+                          example: 1665588176616
+                        modificationTime:
+                          type: integer
+                          example: 1665590392293
+                        usedBytes:
+                          type: integer
+                          example: 943718400
+                        usedNamespace:
+                          type: integer
+                          example: 40000
+                        quotaInBytes:
+                          type: integer
+                          example: 1073741824
+                        quotaInNamespace:
+                          type: integer
+                          example: 50000
+                        owner:
+                          type: string
+                          example: testuser
+                        bucketLayout:
+                          type: string
+                          example: OBJECT_STORE
+                        acls:
+                          type: array
+                          items:
+                            type: object
+                            properties:
+                              type:
+                                type: string
+                                example: USER
+                              name:
+                                type: string
+                                example: testuser
+                              scope:
+                                type: string
+                                example: ACCESS
+                              aclList:
+                                type: array
+                                items:
+                                  type: string
+                                  example: WRITE
+              example:
```

Comment:
We do not require a separate example section, as it will automatically pick up the values from the schema examples